src/assets/scss/_variables.scss

```diff
@@ -138,3 +138,11 @@ $box-shadow-popover: 0 5px 10px rgba(0, 0, 0, 0.2);
 
 // Figma Variables
 $m3-elevation-light-3: 0px 4px 8px 3px rgba(0, 0, 0, 0.15), 0px 1px 3px 0px rgba(0, 0, 0, 0.30);
+
+// GeojsonMap based on Webview Variables
```

Comment:
Is "Webview" a term from Web Forms? How about describing these as "based on Web Forms Variables"?

Reply:
Not a real term, just a typo :D